Makefile

```diff
@@ -0,0 +1,157 @@
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
+
+#	The following lines tell Pe and Eddie where the SRCS, RDEFS, and RSRCS are
+#	so that Pe and Eddie can fill them in for you.
+#%{
+# @src->@ 
+
+#	Specify the source files to use. Full paths or paths relative to the 
+#	Makefile can be included. All files, regardless of directory, will have
+#	their object files created in the common object directory. Note that this
+#	means this Makefile will not work correctly if two source files with the
+#	same name (source.c or source.cpp) are included from different directories.
+#	Also note that spaces in folder names do not work well with this Makefile.
+SRCS = \
+	 bemobile.cpp  \
+	 callregslotview.cpp  \
+	 calview.cpp  \
+	 dialabout.cpp  \
+	 dialcalendar.cpp  \
+	 dialcalevent.cpp  \
+	 dialeditpbook.cpp  \
+	 dialnewsms.cpp  \
+	 gsm.cpp  \
+	 initwindow.cpp  \
+	 main.cpp  \
+	 mainwindow.cpp  \
+	 Matcher.cpp  \
+	 mobileview.cpp  \
+	 Pattern.cpp  \
+	 pbbynameview.cpp  \
+	 pbview.cpp  \
+	 phonelist.cpp  \
+	 phonelistslotview.cpp  \
+	 santa/BetterScrollView/BetterScrollView.cpp  \
+	 santa/ColumnListView/CLVColumnLabelView.cpp  \
+	 santa/ColumnListView/CLVEasyItem.cpp  \
+	 santa/ColumnListView/CLVListItem.cpp  \
+	 santa/ColumnListView/ColumnListView.cpp  \
+	 santa/Cursors/Cursors.cpp  \
+	 santa/NewStrings/NewStrings.cpp  \
+	 santa/PrefilledBitmap/PrefilledBitmap.cpp  \
+	 santa/ScrollViewCorner/ScrollViewCorner.cpp  \
+	 smsboxview.cpp  \
+	 smsinboxview.cpp  \
+	 smsview.cpp  \
+	 statusview.cpp  \
+	 workview.cpp  \
+
+
+#	Specify the resource definition files to use. Full or relative paths can be
+#	used.
+RDEFS = \
+	 Resource.rdef  \
+
+
+#	Specify the resource files to use. Full or relative paths can be used.
+#	Both RDEFS and RSRCS can be utilized in the same Makefile.
+RSRCS = \
+
+
+# End Pe/Eddie support.
+# @<-src@ 
+#%}
+
+#%}
+
+#	Specify libraries to link against.
+#	There are two acceptable forms of library specifications:
+#	-	if your library follows the naming pattern of libXXX.so or libXXX.a,
+#		you can simply specify XXX for the library. (e.g. the entry for
+#		"libtracker.so" would be "tracker")
+#
+#	-	for GCC-independent linking of standard C++ libraries, you can use
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  be device $(STDCPPLIBS)
+
+#	Additional paths paths to look for local headers. These use the form
+#	#include "header". Directories that contain the files in SRCS are
+#	automatically included.
+LOCAL_INCLUDE_PATHS = \
+	 regexp  \
+	 santa/ColumnListView \
+	 santa/Cursors  \
+	 santa/NewStrings  \
+	 santa/PrefilledBitmap \
+	 santa/ScrollViewCorner  \
+	 santa  \	
```

Comment:
Sir, this was the only way it worked. So I think that we shouldn't mess around it. I will fix the tab problem btw.

Reply:
Okay.
That said, I think a few cosmetic edits would be welcome because some lines have 2 white spaces before the backslash, and some have 1.
Maybe it would be worth doing some cleaning and polishing the style of the Makefile.
But since I'm not the owner of the repo I'll let you do whatever you think is better for your PR to be accepted.